cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -21,7 +21,7 @@
     <ul class="o-secondary-navigation_list
                o-secondary-navigation_list__parents">
         {%- for item in items %}
-            <li>
+            <li data-nav-is-active="{{ is_nav_active(request.url, item) }}">
```

Comment:
thanks @kurtw , fixed in https://github.com/cfpb/cfgov-refresh/pull/1937/commits/fd07a9eb4174e564dfa9e3f232c4661b0c657e6f